server/sonar-web/src/main/js/apps/organizations/store/actions.js

```diff
@@ -18,19 +18,29 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 // @flow
```

Comment:
the way data loaded is not related to the store, please move this file outside the `store` directory